inlang/source-code/create-project/src/tryAutoGenProjectConfig.ts

```diff
@@ -5,7 +5,7 @@ import { getLanguageFolderPath } from "./getLanguageFolderPath.js"
 import type { Plugin } from "@inlang/plugin"
 import { Value } from "@sinclair/typebox/value"
 
-// FIXME: get latest major version instead
```

Comment:
[IN THE FUTURE; ignore now] @janfjohannes you can use the `@inlang/marketplace-registry` to get module links to install given that @floriandwt implements a function `getLatestVersion`